cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -107,10 +108,10 @@ export class PassportAuth implements Auth {
    *
    * @param passport - passport.js instance
    */
-  protected setup(passport: passport.Passport) {
-    passport.use(new Strategy(defaultStrategy(this.userSec)));
-    passport.serializeUser(defaultSerializeUser);
-    passport.deserializeUser(defaultDeserializeUser);
+  protected setup(passportApi: passport.Passport) {
+    passportApi.use(new Strategy(defaultStrategy(this.userSec)));
+    passportApi.serializeUser(defaultSerializeUser);
+    passportApi.deserializeUser(defaultDeserializeUser);
```

Comment:
It's my fault :) 
Basically it was shadowing exactly the same value available globally. 
What we trying to do here is to deliver this as parameter so users will not need to require original passport (we just preparing this method to be extended.